CONTRIBUTING.md

```diff
@@ -1,108 +1,136 @@
+     origin  https://github.com/johndoe/react-native-macos.git (push)
+```
 
-You can learn more about the contribution process in the following documents:
+- Map the primary **react-native-macos** repository as the upstream remote
 
-* [Issues](https://github.com/facebook/react-native/wiki/Issues)
-* [Pull Requests](https://github.com/facebook/react-native/wiki/Pull-Requests)
+```bash
+git remote add upstream https://github.com/microsoft/react-native-macos.git
+```
 
-We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).
+- Now running `git remote -v` should show the upstream repository also
 
-### Repositories
+```bash
+git remote -v
 
-The main repository, <https://github.com/facebook/react-native>, contains the React Native framework itself and it is here where we keep track of bug reports and manage pull requests.
+     origin  https://github.com/johndoe/react-native-macos.git (fetch)
+     origin  https://github.com/johndoe/react-native-macos.git (push)
+     upstream        https://github.com/microsoft/react-native-macos.git (fetch)
+     upstream        https://github.com/microsoft/react-native-macos.git (push)
+```
 
-There are a few other repositories you might want to familiarize yourself with:
+- At this point you are ready to start branching and contributing back changes.
 
-* **React Native website** which contains the source code for the website, including the documentation, located at <https://github.com/facebook/react-native-website>
-* **Releases** are coordinated through the <https://github.com/react-native-community/react-native-releases> repository. This includes important documents such as the Changelog.
-* **Discussions** about the future of React Native take place in the <https://github.com/react-native-community/discussions-and-proposals> repository.
-* **High-quality plugins** for React Native can be found throughout the [React Native Community GitHub Organization](http://github.com/react-native-community/).
+### Setting up the branch
 
-Browsing through these repositories should provide some insight into how the React Native open source project is managed.
+For each bug or task you complete, it is recommended that you start with a fresh branch. If you have any lingering changes in your current branch that you want to save, go ahead and commit them. If you are just beginning, then you are good to go. On github, navigate to your repository which should be forked from microsoft/react-native-macos as described in the above sections. Above the list of files is a dropdown that should say master. Use the dropdown to create a new branch and name is according to what you will be working on. (I.e. DropdownHighlight, CleanUpExamples, etc). Now you have created a new branch. 
```

Comment:
Make bold: **microsoft/react-native-macos** to be consistent with other instances.

Reply:
Done.